src/components/useStore/useStore.js

```diff
@@ -3,26 +3,31 @@ import { persist } from 'zustand/middleware';
 import { nanoid } from 'nanoid';
 
 const useStore = create(
-	persist(
-		set => ({
-			newBooks: [],
-			addNewBook: books =>
-				set(state => ({
-					newBooks: [
-						...state.newBooks,
-						{
-							id: nanoid(),
-							author: books.author,
-							title: books.title,
-							comment: books.comment,
-							selection: books.selection,
-							series: books.series,
-						},
-					],
-				})),
-		}),
-		{ name: 'Readembr' }
-	)
+	persist(set => ({
+		newBooks: [],
+		addNewBook: books =>
+			set(state => ({
+				newBooks: [
+					...state.newBooks,
+					{
+						id: nanoid(),
+						author: books.author,
+						title: books.title,
+						comment: books.comment,
+						selection: books.selection,
+						series: books.series,
+					},
+				],
+			})),
+
+		deleteBooks: id => {
+			set(state => {
+				return {
+					newBooks: state.newBooks.filter(books => books.id !== id),
+				};
+			});
+		},
+	}))
 );
```

Comment:
good job! you could call this function "deleteBook" because it always deletes one book.